.vscode/launch.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
not sure if it should be part of this PR.
You can leave it however if it is in separate commit.